tests/deal.II/error_estimator_02.cc

```diff
@@ -241,7 +241,7 @@ void output(const std::string          name,
                            std::string("error"));
   data_out.build_patches ();
 
-  std::ofstream output (name);
+  std::ofstream output (name.c_str());
```

Comment:
Yea. One of the things I've shaken my head about since 1998. It's beyond me how they did not see this one when the first standard was written :-(